src/foam/u2/wizard/DaoWAO.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard',
+  name: 'DaoWAO',
```

Comment:
By the coding-standard, it should be called DAOWAO. All letters of an acronym are treated like one letter, so when capitalized, all should be.

Reply:
How would `DAOWAO` get correctly converted into camelCase or snake_case?

Reply:
Think it would it end up being `daowao`